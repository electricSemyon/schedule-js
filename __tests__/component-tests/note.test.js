import React from 'react';
import renderer from 'react-test-renderer';

import Note from '../../src/components/note.js'

test('renders note div', () => {
  const component = renderer.create(
    <Note body="Test note" id="1" day="1" />
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})
