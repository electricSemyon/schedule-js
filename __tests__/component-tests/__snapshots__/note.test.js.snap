exports[`test renders note div 1`] = `
<div
  className="note active">
  <div
    className="id" />
  <div
    className="body">
    Test note
  </div>
</div>
`;
